.circleci/config.yml

```diff
@@ -296,6 +305,13 @@ workflows:
           requires:
             - deploy-ios
             - deploy-android
+      - push-pods:
+          <<: *release-tags
+          # Xcode 14 not supported until https://github.com/CocoaPods/CocoaPods/issues/11558 is fixed.
```

Comment:
Actually we should be able to use `14.3` since I did #411. This issue was about `watchOS` but that's why I removed it so we could finally move everything here to Xcode 14 :)